HW02/Alexey_Zalyotov/alexey_zalyotov_hw_02_t_02.rb

```diff
@@ -0,0 +1,25 @@
+DATE_TIME_REGEX = %r{\[[[:alnum:]\/\s+:]+\]}.freeze
+IP_REGEX = /[\d.]{7,15}/.freeze
+REQ_PATH_REGEX = %r{[[:upper:]]{3,7}\s[\/[:lower:]\d]+}.freeze
+LINE_REGEX = %r{
+  [\d.]{7,15}               # ip
+  \s-\s-\s                  # separator
+  \[[[:alnum:]\/\s+:]+\]    # datetime
```

Comment:
Can you use CONSTANTs inside your regexp to avoid duplication? 🤔 Something like this
```suggestion
LINE_REGEX = %r{
  #{IP_REGEX}
  \s-\s-\s                  # separator
  #{DATE_TIME_REGEX} 
```

Reply:
Done